src/callbacks_step/summary.jl

```diff
@@ -193,6 +194,18 @@ function initialize_summary_callback(cb::DiscreteCallback, u, t, integrator)
   summary_box(io, "Time integration", setup)
   println()
 
+  # technical details
+  setup = Pair{String,Any}[
+           "#threads" => Threads.nthreads(),
+          ]
+  if mpi_isparallel()
+    push!(setup,
+      "#MPI ranks" => mpi_nranks(),
+    )
+  end
+  summary_box(io, "Technical details", setup)
```

Comment:
Minor remark: "Technical details" sounds very generic, almost like "here goes anything that doesn't fit above". In my previous life, I put such information under the heading "environment" or "environment information" (or similar), to indicate that these are settings that are effected in the environment where Julia is executed (e.g., via the command line, via `ENV` etc.), and sometimes contained additional info such as the hostname, the user name etc., or date&time (which comes in handy if you have 30 stdout.log files lying around).

But we can also leave it as it is if you like "technical details" 👍

Reply:
Caught me :sweat_smile:
I changed it to "Environment information"